utilities/db_util.py

```diff
@@ -1,36 +1,36 @@
-from dotenv import dotenv_values
+import os
+import logging
 import mysql.connector
 
-def connect_to_db(db: str=""):
-  config = dotenv_values(".env")
+def connect_to_db(db: str = ""):
   try:
     db_config = { 
       'host': 'localhost', 
-      'user': config['USER'], 
-      'password': config['PASSWORD']
+      'user': os.getenv('USER'),
+      'password': os.getenv('PASSWORD')
```

Comment:
`application.py` will import them as env variables, we don't need `dotenv_values`